src/algebraic_topology/simplicial_object.lean

```diff
@@ -26,6 +31,29 @@ This is the category of contravariant functors from `simplex_category` to `C`. -
 def simplicial_object := simplex_categoryᵒᵖ ⥤ C
 
 namespace simplicial_object
+
+instance {J : Type v} [small_category J] [has_limits_of_shape J C] :
+  has_limits_of_shape J (simplicial_object C) :=
+let E : (simplex_categoryᵒᵖ ⥤ C) ≌ (ulift.{v} simplex_category)ᵒᵖ ⥤ C :=
+  ulift.equivalence.op.congr_left in
+  adjunction.has_limits_of_shape_of_equivalence E.functor
+
+instance [has_limits C] : has_limits (simplicial_object C) :=
+let E : (simplex_categoryᵒᵖ ⥤ C) ≌ (ulift.{v} simplex_category)ᵒᵖ ⥤ C :=
+  ulift.equivalence.op.congr_left in
+  adjunction.has_limits_of_equivalence E.functor
```

Comment:
```suggestion
instance [has_limits C] : has_limits (simplicial_object C) := 
{ has_limits_of_shape := by apply_instance }
```

In fact, in `has_limits_of_shape` we could add the `auto_param` `. apply_instance`, and then here and similar places where shapes are constructed first we could probably get away with `:= {}`.

Reply:
I made this change.
But changing `has_limits_of_shape` touches many more files and I'm not sure if anything will break (it probably wont), so I would prefer that be left to a separate PR.

Reply:
Okay, I've done this in #6696, and will try to remember to do this one once this PR merges.